src/Maths/Silk.NET.Maths/Scalar.As.cs

```diff
@@ -24,11 +24,11 @@ public partial class Scalar
         /// <typeparam name="TTo">The type converted into</typeparam>
         /// <returns>The converted value</returns>
         [MethodImpl(MaxOpt)]
-        public static TTo As<TFrom, TTo>(TFrom val) where TTo:unmanaged where TFrom:unmanaged
+        public static TTo As<TFrom, TTo>(TFrom val) where TTo : unmanaged where TFrom : unmanaged
         {
             if (typeof(TFrom) == typeof(Half) && typeof(TTo) == typeof(Half))
             {
-                return (TTo)(object)(Half)(Half)(object)val;
+                return (TTo) (object) (Half) (Half) (object) val;
```

Comment:
I don't think I like that :eyes:

Reply:
This is the Silk.NET style. With regards to the double half cast, that's just Alt+Select goodness

Reply:
fair enough, I just made an exception to this file, since it's basically just casts.